app/controllers/users/sessions_controller.rb

```diff
@@ -1,11 +1,19 @@
 # frozen_string_literal: true
 
 class Users::SessionsController < Devise::SessionsController
+  class EmailNotFoundError < StandardError; end
+
   def new
     @email = params[:email]
     super
   end
 
+  def create
+    super
+  rescue EmailNotFoundError
+    render :email_not_found
+  end
```

Comment:
Are school coordinators the only people we expect to stumble onto this page without being registered?

If not, we might need to worry about distinguishing who tried to log in. Definitely not something we want solved in this PR though.

Reply:
Yes, most, if not all, users will use the same passwordless login. There'll be a lot of conditional template rendering for sure but that can be looked at in future pr's. Probably when we start getting content.